label_sync/single_repo_example.yaml

```diff
@@ -0,0 +1,8 @@
+Org: kubernetes
+Repos:
```

Comment:
That file is not needed - it is just a helper to get newest Kubernetes labels file (for example when we need a fresh copy of labels given day to update our `labels.yaml` file).
This file is generated by Marshalling data into YAML (and fields are an upper case because they need to be public):
```
// RepoList is a list of Repositories from Org to be checked by this program
type RepoList struct {
        Org   string
        Repos []github.Repo
}
```
If they were a lower case they won't be accessible in go code then.
So without messing with this and adding complexity to the code, I will leave it "as is".
Other option can be to remove that file totally. It is not used for normal workflow. It is only a local mode helper if You want to get newest label's set from a given org/repo list.